ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -612,6 +612,17 @@ let remove_legacy_ssl_support = {
   fn=fun ~__context -> Db.Host.get_all ~__context |> List.iter (fun self -> Db.Host.set_ssl_legacy ~__context ~self ~value:false)
 }
 
+let update_xapi_version = {
+  description = "Bump API version";
+  version=(fun _ -> true);
+  fn=fun ~__context ->
+    Db.Host.get_all ~__context |>
+    List.iter (fun self ->
+      Db.Host.set_API_version_major ~__context ~self ~value:(Int64.of_int Constants.version_major);
```

Comment:
What happens during an RPU when the master got upgraded, but the slave hasn't yet? This looks more like a pool version than a host version now.
If this causes other problems could we set the API version on toolstack startup just for the current host? That'd give a better picture...

Reply:
Dbsync_slave refresh_localhost_info already has some code to do this refresh (and apparently runs on master too).
Is the problem that the import code runs too soon, before that version is updated?